lnwallet/interface_test.go

```diff
@@ -1238,6 +1237,7 @@ var walletTests = []func(miner *rpctest.Harness, w *lnwallet.LightningWallet, te
 	testTransactionSubscriptions,
 	testListTransactionDetails,
 	testSignOutputPrivateTweak,
+	testCancelNonExistantReservation,
```

Comment:
Nice catch! I think one of my prior commits adding a new test-case accidentally removed this test.